pkg/imagestore/imagestore.go

```diff
@@ -287,25 +290,31 @@ func (s *rhcosStore) Populate(ctx context.Context) error {
 		if arch == "s390x" {
 			continue
 		}
-		minimalPath := filepath.Join(s.dataDir, isoFileName(ImageTypeMinimal, openshiftVersion, imageVersion, arch))
-		if _, err := os.Stat(minimalPath); os.IsNotExist(err) {
-			log.Infof("Creating minimal iso for %s-%s-%s", openshiftVersion, imageVersion, arch)
+		grp.Go(func() error {
+			minimalPath := filepath.Join(s.dataDir, isoFileName(ImageTypeMinimal, openshiftVersion, imageVersion, arch))
+			if _, err := os.Stat(minimalPath); os.IsNotExist(err) {
+				log.Infof("Creating minimal iso for %s-%s-%s", openshiftVersion, imageVersion, arch)
 
-			fullPath := filepath.Join(s.dataDir, isoFileName(ImageTypeFull, openshiftVersion, imageVersion, arch))
-			rootfsURL, err := buildRootfsURL(s.imageServiceBaseURL, arch, openshiftVersion)
-			if err != nil {
-				return fmt.Errorf("failed to build rootfs URL: %v", err)
-			}
+				fullPath := filepath.Join(s.dataDir, isoFileName(ImageTypeFull, openshiftVersion, imageVersion, arch))
```

Comment:
Not relevant to this PR, but it looks like we're processing everything on the PV. I am not familiar with the internals of building the image: is the process more I/O intensive than just copying something from memory to the disk? Asking because if it is, we might benefit from working in tmpfs (and maybe to some extent also in ephemeral storage) and copying the final result. @carbonin thoughts?